File: Programming Basics/JavaScript/Introduction/Labs/yard-greening.js
function solve(input) {
    const pricePerSquareMeter = 7.61;
    const discountPercentage = 0.18;

    const squareMeters = Number(input[0]);

    const totalCosts = squareMeters * pricePerSquareMeter;
    const discount = discountPercentage * totalCosts;
    const finalPrice = totalCosts - discount;

    console.log(`The final price is: ${finalPrice} lv.`);
    console.log(`The discount is: ${discount} lv.`);
}

File: Programming Basics/C++/Introduction/Labs/YardGreening/YardGreening.cpp
#include <iostream>
using namespace std;

int main()
{
    const double pricePerSquareMeter = 7.61;
    const double discountPercentage = 0.18;

    double squareMeters;
    cin >> squareMeters;

    double totalCosts = squareMeters * pricePerSquareMeter;
    double discount = discountPercentage * totalCosts;
    double finalPrice = totalCosts - discount;

    cout << "The final price is: " << finalPrice << " lv." << endl;
    cout << "The discount is: " << discount << " lv." << endl;
}
